migration/migrator/migrations/course/20250312145730_add_redactions.py

```diff
@@ -0,0 +1,44 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("""
+                     CREATE TABLE IF NOT EXISTS gradeable_redaction (
+                     redaction_id SERIAL PRIMARY KEY,
+                     g_id character varying(255) NOT NULL,
+                     page integer NOT NULL,
+                     x1 float NOT NULL,
+                     x2 float NOT NULL,
+                     y1 float NOT NULL,
+                     y2 float NOT NULL,
+                     FOREIGN KEY (g_id) REFERENCES gradeable(g_id) ON DELETE CASCADE
```

Comment:
You can simplify this by adding `REFERENCES ...` to the end of the `g_id` definition.